scripts/nfs-ganesha/pull-sign-push

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# The script runnings on the signer box will pull nfs-ganesha packags that wore created in the last 24 hours on chacra.ceph.com to /opt/new-repos
+# After this the sign-rpms-auto script will run and sign the nfs-ganesha packages
+# And finally the sync-push-auto script will run and will push the signed packages to download.ceph.com
+
+today_items=$(ssh ubuntu@chacra.ceph.com 'find /opt/repos/nfs-ganesha-stable -newermt "-24 hours" -ls' | awk '{ print $11 }' )
+if [ -n "$today_items" ]; then
+echo "pulling nfs-ganesha packages from chacra"
+echo "********************************************"
+[[ -d /opt/new-repos/ ]] | mkdir -p /opt/new-repos/
+  for item in $today_items; do
+   sync_cmd="ubuntu@chacra.ceph.com:$item /opt/new-repos/"
```

Comment:
Can you make this something like `/opt/nfs-ganesha/new-repos` so it's clear what sort of repos they are?

Reply:
Yes changed the folder name